### PR TITLE
Add 'this is a pop-up clinic' warning to PharmacyCard

### LINKED ISSUE #106: Add 'this is a pop-up' warning

### PR DESCRIPTION
**Pull request by luceliu**
Closes #106 

`K2T`: 
image

### CODE REVIEW

File: src/containers/PharmacyList/PharmacyList.tsx
Comment (chrisahardie) [inline src/containers/PharmacyList/PharmacyList.tsx:181]: Why are we defaulting to 1?

File: src/components/PharmacyCard/PharmacyCard.tsx
Comment (chardielxr) [inline src/components/PharmacyCard/PharmacyCard.tsx:38]: Unless I'm mistaken, organization is optional - as such this should be marked optional with `organizationId?: number`

Comment (chrisahardie) [inline src/components/PharmacyCard/PharmacyCard.tsx:38]: Unless I'm mistaken, organizations are optional - if so, this should be an optional prop: `organizationId?: number`

File: src/components/PharmacyCard/PharmacyCard.tsx
Comment (chrisahardie) [inline src/components/PharmacyCard/PharmacyCard.tsx:71]: You would need to update this to `organizationId: number | undefined`

I like the contextual information provided in the function name, though I personally use constants. The function is also pretty tiny, and could be written inline:

```
const POPUP_ORGANIZATION_ID = 25;
const popupWarningMarkup =props.organizationId === POPUP_ORGANIZATION_ID ? (
```

File: src/components/PharmacyCard/PharmacyCard.tsx
Comment (andrewpmk) [inline src/components/PharmacyCard/PharmacyCard.tsx:75]: Please translate with i18next - use t() function. We can add to the translation spreadsheet.

**Comment by andrewpmk** [comment]
I added popupwarning and popupwarningmessage to the translations spreadsheet.

My French translations:

popupwarning: Ceci est une clinique sans rendez-vous
popupwarningmessage: Veuillez visiter le site Web pour confirmer que vous êtes éligible pour ce site de vaccination.

File: src/locales/en/translation.json
Comment (andrewpmk) [inline src/locales/en/translation.json:27]: Should be translated into other languages.

Comment (chrisahardie) [reply src/locales/en/translation.json:27]: Are we going to wait until we have the translations before we merge, or can we merge and cut a new ticket to track that work?

Comment (andrewpmk) [reply src/locales/en/translation.json:27]: it's not a big deal, would you like me to update the branch 106-add-popup-warning to add the French? Just make sure to pull and merge conflicts afterwards.

Comment (andrewpmk) [reply src/locales/en/translation.json:27]: added the French translations

Comment (luceliu) [reply src/locales/en/translation.json:27]: we should start a new ticket for the rest of the translations

Comment (andrewpmk) [reply src/locales/en/translation.json:27]: Created a ticket https://github.com/Vaccine-Hunters-Canada/VaxFinder-Frontend/issues/110

**Review by ericherscovich — APPROVED** [review]

**Pull request merged**
